Libs/RaptureCore/src/main/java/rapture/kernel/AuditApiImpl.java

```diff
@@ -65,7 +65,7 @@
  * 
  */
 public class AuditApiImpl extends KernelBase implements AuditApi {
-    private static final RaptureURI KERNEL_URI = RaptureURI.builder(Scheme.LOG, "//kernel").build();
+    private static final RaptureURI KERNEL_URI = RaptureURI.builder(Scheme.LOG, "kernel").build();
```

Comment:
Because when I delved into the audit log I realized that it was creating entries for both log://kernel and log:///kernel